base/daemon/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ add_library (daemon
 )
 
 target_include_directories (daemon PUBLIC ..)
+
+if (OS_DARWIN AND NOT MAKE_STATIC_LIBRARIES)
+    target_link_libraries (daemon PUBLIC -Wl,-undefined,dynamic_lookup)
```

Comment:
I don't understand the syntax `-undefined,dynamic_lookup` - why `dynamic_lookup` is without `-` and what does it mean?

Ok, don't mind.

Reply:
Ok, I understand that it will be passed to linker as:
`-undefined dynamic_lookup`